### PR TITLE
added roadmap

### PR DESCRIPTION
**Pull request by ariel-zilber**
Hi i think that the docs are missing a a simple section for your roadmap
I added such section
Cheers,

### CODE REVIEW

**Comment by ariel-zilber** [comment]
Can someone  please check it😊

**Comment by gcolliso** [comment]
Reviewing - thanks @ariel-zilber

File: running-a-nats-service/configuration/mqtt/README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ If the MQTT subscription is QoS1 and an MQTT publisher publishes an MQTT QoS1 me
 When the server delivers a QoS 1 message to a QoS 1 subscription, it will keep the message until it receives the PUBACK for the corresponding packet identifier. If it does not receive it within the "ack_wait" interval, that message will be resent.
 
 ## Max Ack Pending
-
+z
```
Comment (gcolliso) [inline running-a-nats-service/configuration/mqtt/README.md:67]: This should be removed and not updated unless you had some other correction in mind.

File: docs/index.html [hunk 2]
```diff
@@ -137,8 +137,20 @@
 
             
         </li>
-    
-        <li class="chapter " data-level="1.4" >
+        <li class="chapter " data-level="1.3" data-path="faq.html">
```
Comment (gcolliso) [inline docs/index.html:140]: This file is not used in production - please remove this change.

Comment (ariel-zilber) [reply docs/index.html]: fixed :)

File: running-a-nats-service/configuration/mqtt/README.md (hunk 1 above)
Comment (ariel-zilber) [reply running-a-nats-service/configuration/mqtt/README.md]: fixed:)

**Review by gcolliso — APPROVED** [review]
lgtm - thanks @ariel-zilber

**Pull request merged**